client/hotrod-client/src/test/java/org/infinispan/client/hotrod/ExecTest.java

```diff
@@ -41,6 +50,11 @@ protected void createCacheManagers() throws Throwable {
       createHotRodServers(NUM_SERVERS, new ConfigurationBuilder());
    }
 
+   @Test(expectedExceptions = HotRodClientException.class, expectedExceptionsMessageRegExp = ".*ISPN027002.*")
```

Comment:
I would prefer not to use the numbers here. They might change over time (or in the product) and this test might start failing. Can we use some specific phrase in the exception?